apps/src/lib/ui/PopUpMenu.jsx

```diff
@@ -116,46 +120,84 @@ class MenuBubbleUnwrapped extends Component {
       <div style={style} className={className}>
         {this.renderMenuItems()}
         {/* These elements are used to draw the 'tail' with CSS */}
-        <span style={tailBorderStyle}/>
-        <span style={tailFillStyle}/>
+        {!this.props.minimalStyle &&
+          <span style={tailBorderStyle}/>
+        }
+        {!this.props.minimalStyle &&
+          <span style={tailFillStyle}/>
+        }
       </div>
     );
   }
 }
 export const MenuBubble = Radium(MenuBubbleUnwrapped);
 
+class MenuBreak extends Component {
+
+  render() {
+    const style = {
+      borderTop: '1px solid ' + color.lighter_gray,
+      marginTop: STANDARD_PADDING/2,
+      marginBottom: STANDARD_PADDING/2,
+      marginLeft: STANDARD_PADDING,
+      marginRight: STANDARD_PADDING,
+    };
+    return (
+      <div style={style}></div>
+    );
+  }
+}
+
+PopUpMenu.MenuBreak = Radium(MenuBreak);
+
 class Item extends Component {
   static propTypes = {
-    children: PropTypes.string.isRequired,
+    children: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.array]).isRequired,
     onClick: PropTypes.func,
+    href: PropTypes.string,
     first: PropTypes.bool,
     last: PropTypes.bool,
+    color: PropTypes.string,
+    heightDivisor: PropTypes.number,
```

Comment:
This part in particular would be great to not have.

Reply:
Reading this props list without the component implementation, I don't really know what `heightDivisor` does.  At least having a comment documenting this prop would be nice, so someone wanting to reuse `Item` wouldn't need to read its whole implementation.